components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/add.jsp

```diff
@@ -444,11 +445,18 @@
                                 <tr>
                                     <td class="leftCol-med"><fmt:message key='oauth.version'/><span
                                             class="required">*</span></td>
+                                    <%
+                                    boolean isOAuth1Enabled = Boolean.parseBoolean(IdentityUtil.getProperty("LegacyFeatures.EnableOAuth1"));
+                                    if (!isOAuth1Enabled) {
```

Comment:
shouldn't we prevent displaying the input field if if isOAuth1Enabled not enabled?

Reply:
Corrected https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1407/commits/e5dca5f8761be050156782479f65a2d611f76780